service/src/test/java/uk/nhs/adaptors/gp2gp/ehr/mapper/EhrExtractMapperTest.java

```diff
@@ -80,11 +89,26 @@ public static void initialize() throws IOException {
             .build();
     }
 
+    @AfterAll
+    public static void deinitialize() {
+        TimeZone.setDefault(null);
```

Comment:
Setting/resetting the default timezone should not be required. Either remove here, or ensure that https://gpitbjss.atlassian.net/browse/NIAD-1060 removes them

Reply:
Removal of these methods will cause different timings to be asserted when run in different time zones.